app/javascript/api/adapters/listing/ListingPropertyAdapter.ts

```diff
@@ -0,0 +1,39 @@
+import { Property } from "@bloom-housing/backend-core/types"
+
+import RailsRentalListing from "../../types/rails/listings/RailsRentalListing"
+import { Adapter } from "../adapter"
+import ListingAddressAdapter from "./ListingAddressAdapter"
+import UnitSummariesAdapter from "./UnitSummariesAdapter"
+
+const getMinAndMaxHouseholdSize = (_listing: RailsRentalListing): [number, number] | undefined =>
+  // todo: implement
+  [1, 5]
+
+const ListingPropertyAdapter: Adapter<RailsRentalListing, Property> = (
+  listing: RailsRentalListing
+) => {
+  const [minHousehold, maxHousehold] = getMinAndMaxHouseholdSize(listing)
+  return {
+    unitsSummarized: UnitSummariesAdapter(listing),
+    units: [], // todo: populate this field
```

Comment:
we'll need this more detailed unit info on the listing details page, but we don't need it for the directory so we can skip it for now